libraries/cbor/Cargo.toml

```diff
@@ -14,5 +14,9 @@ homepage = "https://github.com/google/OpenSK"
 repository = "https://github.com/google/OpenSK"
 keywords = ["cbor", "serialization", "no_std"]
 categories = ["encoding"]
+readme = "README.md"
+
+[badges]
```

Comment:
Badges are deprecated now: https://doc.rust-lang.org/cargo/reference/manifest.html#the-badges-section. Instead we should add them in the README directly (or write some text).

Reply:
I see that as complementary.
The badges section is not mentioned as being deprecated. Only the automatic display is.
I guess that the maintenance status from the manifest is still being used by, for example, `cargo audit`.

I can see a few badges which may be nice to put in our readme.md file though:
Number of downloads:
`[![crates.io](https://img.shields.io/crates/d/sk-cbor.svg)](https://crates.io/crates/sk-cbor)`

Latest published version:
`[![crates.io](https://img.shields.io/crates/v/sk-cbor.svg)](https://crates.io/crates/sk-cbor)`

Latest doc version:
`[![docs.rs](https://docs.rs/sk-cbor/badge.svg)](https://docs.rs/sk-cbor)`

License:
`[![License](https://img.shields.io/crates/l/sk-cbor.svg)](https://crates.io/crates/sk-cbor)`

Regarding maintenance badges the only thing I could find is only a self-declarative boolean:
`[![Maintenance](https://img.shields.io/maintenance/yes/2021)](https://crates.io/crates/sk-cbor)`

Reply:
Good ideas, I added them.